src/System.Private.Xml/tests/System.Private.Xml.Tests.builds

```diff
@@ -3,65 +3,125 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="Readers\CharCheckingReader\System.Xml.RW.CharCheckingReader.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
       <OSGroup>Windows_NT</OSGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
+    <Project Include="Readers\CharCheckingReader\System.Xml.RW.CharCheckingReader.Tests.csproj">
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
I see that in all of the configurations in this .builds file we are passing `OSGroup` to be `Windows_NT`. Is that actually required? E.g. Do we actually have windows-specific tests and we only want to run those?